test/dev-server.unit.js

```diff
@@ -104,7 +104,8 @@ test('[DevServer] Does not install builders if there are no builds', async t =>
   process.stdout.addListener('data', handler);
   process.stderr.addListener('data', handler);
 
-  await installBuilders(new Set());
```

Comment:
@TooTallNate There was no yarn (2nd param) being passed to this test so I kept it as undefined.